src/docs/windows/pwa-api-documentation/pwa-api-documentation.md

```diff
@@ -0,0 +1,312 @@
+- Periodic Sync: The Periodic Sync API enables your PWA to sync data in the background (without your PWA being open) if the device is currently online. This enables common scenarios such as:
+  o A news app can ensure the user always has the latest news as soon as they open the app as the data has been synced in the background before the app was opened.
+  o A music app can ensure a user’s library on a device is always in sync with the cloud.
+
+<div class="demo-block">
+  <strong>Demo</strong>
+
+  <p>
+    Check out a PWA <a href="https://webboard.app/" target="_blank" rel="noopener">here</a> that works completely offline! Open the link once so that the app caches its assets, then close it and open it again while on airplane mode, you will notice the app still works fine!
+  </p>
+</div>
+
+<div class="how-to">
+  <strong>How To</strong>
+
+  <p>
+    Check out <a href="https://docs.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/offline#use-the-caching-to-improve-performance-of-pwas" target="_blank" rel="noopener">this tutorial</a> to learn more about making your PWA work offline, <a href="https://developers.google.com/web/updates/2015/12/background-sync" target="_blank" rel="noopener">this tutorial</a> for more on the Background Sync API and finally, <a href="https://web.dev/periodic-background-sync/" target="_blank" rel="noopener">this tutorial</a> on using the Periodic Sync API.
+  </p>
+</div>
+
+## Push Notifications
+PWAs can also receive push notifications, just like any other app on Windows! You will need a Service Worker for this to work (just as with the offline APIs above) and then you can use the Push API to receive push notifications from your server! These notifications will show using the built-in notifications UI in Windows.
+
+<div class="how-to">
+  <strong>How To</strong>
+
+  <p>
+    Check out <a href="https://docs.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/serviceworker#push-notifications" target="_blank" rel="noopener">this tutorial</a> to learn how to set up push notifications for your PWA.
+  </p>
+</div>
+
+## Media Controls and playing Media in the background
+PWAs can tie into the same native media controls as native apps, allowing users to play, pause and skip media while your PWA is in the background using the [Media Session API](https://developer.mozilla.org/en-US/docs/Web/API/Media_Session_API). 
+
+<div class="how-to">
+  <strong>How To</strong>
+
+  <p>
+    Check out <a href="https://developer.mozilla.org/en-US/docs/Web/API/Media_Session_API" target="_blank" rel="noopener">this tutorial</a> to learn how to use the Media Session API in your media PWA!
```

Comment:
The exclamation mark is too over the top for this sentence.

Reply:
Might be helpful to show a screenshot of media session in action. Here are a few if you need any:

Windows will show your PWA's currently playing media with media controls whenever you adjust Windows' volume or press play/pause/stop keys on your keyboard:
![image](https://user-images.githubusercontent.com/312936/130278845-672fb730-9531-4749-a670-21ba277952cf.png)

Your PWA's media with controls shown on the Windows lock screen:
![image](https://user-images.githubusercontent.com/312936/130279103-0faba67c-5513-4831-9f92-7698b9959800.png)